code/game/area/Space Station 13 areas.dm

```diff
@@ -54,6 +54,8 @@ NOTE: there are two lists of areas in the end of this file: centcom and station
 	var/has_gravity = 0
 	var/safe = 0 				//Is the area teleport-safe: no space / radiation / aggresive mobs / other dangers
 
+	var/blacklisted = 0			//Rejects ghosts

```

Comment:
Please give this a better name, it should be descriptive enough that the comment isn't needed.